src/main/kotlin/com/cognifide/gradle/aem/common/instance/local/BackupManager.kt

```diff
@@ -44,10 +44,19 @@ class BackupManager(private val aem: AemExtension) {
         aem.prop.file("localInstance.backup.localDir")?.let { set(it) }
     }
 
+    /**
+     * Determines the number of local backup ZIP files stored.
+     * After creating new backups, old ones are auto-removed.
+     */
+    val localCount = aem.obj.int {
+        convention(2)
```

Comment:
why 2? let's imagine that you have some backup created initially then after months created another one, you want to go back to the initial state and skip the backup that you just created / at the wrong moment or assuming negatively when just created backup is corrupted but initially created not.

Reply:
still this will protect against reserving too much GB infinitely